pkg/api/v1beta3/types.go

```diff
@@ -206,7 +206,10 @@ type HostPathVolumeSource struct {
 	Path string `json:"path" description:"path of the directory on the host"`
 }
 
-type EmptyDirVolumeSource struct{}
+type EmptyDirVolumeSource struct {
+	// Optional: make this empty directory be backed by memory.
+	Tmpfs bool `json:"tmpfs" description:"use tmpfs instead of disk"`
```

Comment:
If we think we will address this before the "freeze" for v1beta3 then the flag makes sense. If not, the type will be easier to extend from. @bgrant0607, your call :)

This PR is otherwise LGTM'd.